lib/bike_brigade/messaging.ex

```diff
@@ -53,6 +53,31 @@ defmodule BikeBrigade.Messaging do
     Repo.all(query)
   end
 
+
+  @doc"""
+  Used to fetch sms conversations from riders who are riding for a specific
+  campaign. It uses a left_lateral_join (as opposed to the inner_lateral_join in
+  `list_sms_conversastions`) because we need to return all riders from the
+  passed in list, even if they have no sms history (albeit unlikely)).
+  """
+  def list_sms_for_campaign(rider_ids) do
```

Comment:
I find this function name confusing.

If it's searching by campaign we should pass it a campaign id and do some more complicated join across the camapaigns table. 

If we want to give it rider ids, we should call it `list_sms_conversations_for_riders` or `list_lasted_sms_messages_for_riders`

Personally i think the campaign ID makes more sense but I haven't thought about it too much

Reply:
I see that campaign_id is nullable (and most rows have `campaign_id` set to `null` in my local db) in the `sms_mesages` table. So, I'm thinking that maybe we just change the name for now?